### PR TITLE
modules/opensles: fix recorder speaker setup

### PR DESCRIPTION
**Pull request by juha-h**
- if there is more than one channel, use SL_SPEAKER_FRONT_LEFT |
  SL_SPEAKER_FRONT_RIGHT speakers

### CODE REVIEW

File: modules/zrtp/zrtp.c [hunk 1]
```diff
@@ -292,7 +292,8 @@ static void on_zrtp_secure(zrtp_stream_t *stream)
 		     (size_t)sess_info.peer_zid.length,
 		     sess_info.peer_zid.buffer,
 		     (size_t)sess_info.peer_zid.length);
-	} else if (sess_info.sas_is_verified) {
+	}
+	else if (sess_info.sas_is_verified) {
```
Comment (alfredh) [inline modules/zrtp/zrtp.c]: this change is not related.

**Comment by alfredh** [comment]
looks very good, except the unrelated change.

**Comment by juha-h** [comment]
Alfred E. Heggestad writes:

> > @@ -292,7 +292,8 @@ static void on_zrtp_secure(zrtp_stream_t *stream)
>  		     (size_t)sess_info.peer_zid.length,
>  		     sess_info.peer_zid.buffer,
>  		     (size_t)sess_info.peer_zid.length);
> -	} else if (sess_info.sas_is_verified) {
> +	}
> +	else if (sess_info.sas_is_verified) {
> 
> this change is not related.

No, it was another commit to make travis happy.

-- Juha

**Comment by alfredh** [comment]
ok. I merged it to master :)

Thanks!
